dm/subtask_master.go

```diff
@@ -156,8 +157,14 @@ func (s *SubTaskMaster) OnWorkerOnline(worker lib.WorkerHandle) error {
 }
 
 func (s *SubTaskMaster) OnWorkerOffline(worker lib.WorkerHandle, reason error) error {
-	log.L().Info("on worker offline")
-	return nil
+	log.L().Info("on worker offline", zap.Any("worker handle", worker.ID()),
```

Comment:
How do we handle the case where the master and a worker crash at the same time?

Reply:
currently this is not supported, this PR only add a test about syncer container failure